web/modules/custom/configform/src/DatabaseService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\configform;
+
+use Drupal\Core\Database\Connection;
+
+/**
+ * Instantaniates database connection and inserts data.
+ */
+class DatabaseService {
```

Comment:
If you have added this service just to create a wrapper function for the database::insert() then this is not required.

Reply:
I don't know how it was previously, but this class is not needed.

Reply:
OK, i'll remove it

Reply:
> I don't know how it was previously, but this class is not needed.

Previously it was :
`\Drupal::database()->insert('configform_example')->fields([
      'email' => $values['email'],
      'name' =>  $values['name'],
      'password' => md5($values['password'])
    ])->execute();`

Reply:
Fixed